.github/workflows/testing.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - { LN_IMPL: clightning }
+        LN_IMPL: [clightning, ldk]
+
     steps:
       - uses: actions/checkout@v2
       - name: Integration testing
         run: |
           docker build -f docker/Dockerfile.${{matrix.LN_IMPL}} -t lnprototest-${{matrix.LN_IMPL}} .
-          docker run lnprototest-${{matrix.LN_IMPL}}
+          docker run -e LN_IMPL=${{matrix.LN_IMPL}} lnprototest-${{matrix.LN_IMPL}}
```

Comment:
why do we need this `LN_IMPL` environment variable?

Reply:
> why do we need this `LN_IMPL` environment variable?

It is getting used in `docker/docker-compose.yml`.

Earlier it was also getting used in `entrypoint.sh` when I made one entrypoint for both ldk and clightning, but I changed it and added `ldk-entrypoint.sh`

Reply:
ah ok make sense thanks!